src/stratigraphy/util/line.py

```diff
@@ -24,6 +24,12 @@ def __init__(self, rect: fitz.Rect, text: str):
     def __repr__(self) -> str:
         return f"TextWord({self.rect}, {self.text})"
 
+    def to_json(self):
+        return {
+            "text": self.text,
+            "rect": [self.rect.x0, self.rect.y0, self.rect.x1, self.rect.y1],
+        }
```

Comment:
is this used at all?

Reply:
Yes it is. When we save the depth column to the json, probably for the LayerIdentifierColumn. In any case, after merging your changes the code didn't run any more since TextWord did not have a .to_json method.

Reply:
You are right. It was only used since add_entry added words to the LayerIdentifierColumn and not LayerIdentifierEntry. Now we manage without the to_json method.